rasa/core/policies/default_flows.yml

```diff
@@ -98,3 +98,23 @@ flows:
         next: "1"
       - id: "1"
         action: utter_clarification_options_rasa
+
+  pattern_ask_question:
+    description: flow used to fill a slot
+    steps:
+      - id: "start"
+        action: action_extract_slots
```

Comment:
I'm assuming this allows the user to continue using existing custom extraction and validation actions from `rasa-sdk`? While the sdk `ValidationAction`still makes sense, how do we adapt away from usages of `FormValidationAction` (used only within form context)?

Reply:
mhm good question, we probably should create a cleaned up validation action specifically for flows